### PR TITLE
move to docker

### PR DESCRIPTION
**Pull request by putstrln**
@JDPelayoPetit @spprashant

### CODE REVIEW

File: server/pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version = "1.0.0"
 description = "API for movie butler app"
 authors = ["Your Name <you@example.com>"]
 readme = "README.md"
-packages = [{include = "movie_butler"}]
+#packages = [{include = "movie_butler"}]
```
Comment (JDPelayoPetit) [inline server/pyproject.toml:7]: Why this change?

**Review by JDPelayoPetit — APPROVED** [review]

File: server/pyproject.toml (hunk 1 above)
Comment (putstrln) [reply server/pyproject.toml:7]: we don't really need it since we won't be publishing anything. It was also making it a bit difficult for docker since we first copy the lock file without the codebase.

Comment (JDPelayoPetit) [reply server/pyproject.toml:7]: Ah gotcha

File: server/docker/test/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: "3"
```
Comment (spprashant) [inline server/docker/test/docker-compose.yml:1]: Newb question, why version 3 here and 3.6 in the other file?

**Review by spprashant — APPROVED** [review]

File: server/docker/test/docker-compose.yml (hunk 2 above)
Comment (putstrln) [reply server/docker/test/docker-compose.yml:1]: i was fixing a syntax issue because something wasn't supported and forgot to put it back.
Fixed
